.github/workflows/commit_check.yaml

```
@@ -1,67 +1,167 @@
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+          cache: 'pip'
+      
+      - name: Cache virtual environment
+        uses: actions/cache@v4
+        id: venv-cache
+        with:
+          path: venv
+          key: venv-${{ runner.os }}-${{ env.PYTHON_VERSION }}-${{ hashFiles('requirements/dev.txt') }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)